src/tests.cpp

```diff
@@ -481,8 +484,9 @@ void tbf(std::string good_examples_file, std::string bad_examples_file) {
     Workload wl(100, 1, total_time);
 
     for (uint i = 0; i < interval; i++) {
+        Comp* comp_spec = new Comp(Indiv(metric_t::CENQ, 0), op_t::GE, (unsigned int)(i + 1) * link_rate);
         wl.add_spec(
-            TimedSpec(Comp(Indiv(metric_t::CENQ, 0), op_t::GE, (unsigned int) (i + 1) * link_rate),
+            TimedSpec(comp_spec,
```

Comment:
Just add a new, no need for a separate variable.

Reply:
[Tests handle pointers more elegantly](https://github.com/all-things-networking/fperf/pull/27/commits/b9bc7fb16bff8b1a55e3f11963048e071ca1ad0b)